Project.toml

```
@@ -5,9 +5,11 @@ version = "0.9.7"
 [deps]
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 MathOptInterface = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
+SCIP_jll = "e5ac4fe4-a920-5659-9bf8-f9f73e9e79ce"
 
 [compat]
 MathOptInterface = "^0.9.6"
+SCIP_jll = "^0.1.2"
 julia = "^1.0.0"
```

```suggestion
julia = "1"
```
is simpler and equivalent